### PR TITLE
chore: add .swcrc to .npmignore (file updates)

### PR DESCRIPTION
**Pull request by layershifter**
_Follows #27739._

## Previous Behavior

image

## New Behavior

`.swcrc` is excluded from publishing. This PR that updates `.npmignore`, changes in a generator happened in #27739.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/20f0a213b94eda2d8371d8e14d2baec875001bf8'>20f0a213b94eda2d8371d8e14d2baec875001bf8</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/Avatar-mount_base.html">697</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/Avatar-mount.html">723</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/Button-mount_base.html">397</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/Button-mount.html">403</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/Field-mount_base.html">1241</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/Field-mount.html">1257</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/FluentProvider-mount_base.html">787</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/FluentProvider-mount.html">806</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/FluentProviderWithTheme-mount_base.html">107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/FluentProviderWithTheme-mount.html">111</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">95</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">98</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">109</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/InfoButton-mount_base.html">23</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/InfoButton-mount.html">26</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/MakeStyles-mount_base.html">1242</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/MakeStyles-mount.html">1189</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/Persona-mount_base.html">1920</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/Persona-mount.html">1871</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/SpinButton-mount_base.html">1470</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test-react-components/SpinButton-mount.html">1471</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/BaseButton-mount_base.html">795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/BaseButton-mount.html">793</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Breadcrumb-mount_base.html">2048</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Breadcrumb-mount.html">2032</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Checkbox-mount_base.html">2088</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Checkbox-mount.html">2101</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/CheckboxBase-mount_base.html">1844</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/CheckboxBase-mount.html">1855</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ChoiceGroup-mount_base.html">3472</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ChoiceGroup-mount.html">3528</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ComboBox-mount_base.html">811</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ComboBox-mount.html">786</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/CommandBar-mount_base.html">7669</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/CommandBar-mount.html">7619</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ContextualMenu-mount_base.html">12205</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ContextualMenu-mount.html">12477</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DefaultButton-mount_base.html">928</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DefaultButton-mount.html">944</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DetailsRow-mount_base.html">2641</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DetailsRow-mount.html">2659</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DetailsRowFast-mount_base.html">2646</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DetailsRowFast-mount.html">2597</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DetailsRowNoStyles-mount_base.html">2409</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DetailsRowNoStyles-mount.html">2410</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Dialog-mount_base.html">3218</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Dialog-mount.html">3243</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DocumentCardTitle-mount_base.html">295</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/DocumentCardTitle-mount.html">288</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Dropdown-mount_base.html">2440</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Dropdown-mount.html">2445</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/FocusTrapZone-mount_base.html">1405</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/FocusTrapZone-mount.html">1382</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/FocusZone-mount_base.html">1395</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/FocusZone-mount.html">1381</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedList-mount_base.html">45198</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedList-mount.html">45359</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedList-virtual-rerender_base.html">22024</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedList-virtual-rerender.html">22079</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">62117</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedList-virtual-rerender-with-unmount.html">61981</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedListV2-mount_base.html">279</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedListV2-mount.html">286</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedListV2-virtual-rerender_base.html">262</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedListV2-virtual-rerender.html">267</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">281</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">292</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/IconButton-mount_base.html">1311</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/IconButton-mount.html">1302</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Label-mount_base.html">431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Label-mount.html">442</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Layer-mount_base.html">3422</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Layer-mount.html">3366</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Link-mount_base.html">517</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Link-mount.html">506</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/MenuButton-mount_base.html">1135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/MenuButton-mount.html">1163</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/MessageBar-mount_base.html">25874</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/MessageBar-mount.html">25796</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Nav-mount_base.html">2355</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Nav-mount.html">2352</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/OverflowSet-mount_base.html">987</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/OverflowSet-mount.html">1007</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Panel-mount_base.html">2195</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Panel-mount.html">2217</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Persona-mount_base.html">901</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Persona-mount.html">922</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Pivot-mount_base.html">1063</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Pivot-mount.html">1062</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/PrimaryButton-mount_base.html">1027</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/PrimaryButton-mount.html">1038</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Rating-mount_base.html">5774</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Rating-mount.html">5743</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/SearchBox-mount_base.html">1119</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/SearchBox-mount.html">1115</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Shimmer-mount_base.html">2358</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Shimmer-mount.html">2311</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Slider-mount_base.html">1672</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Slider-mount.html">1644</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/SpinButton-mount_base.html">3497</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/SpinButton-mount.html">3414</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Spinner-mount_base.html">512</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Spinner-mount.html">518</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/SplitButton-mount_base.html">2189</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/SplitButton-mount.html">2158</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Stack-mount_base.html">538</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Stack-mount.html">537</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/StackWithIntrinsicChildren-mount_base.html">1069</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/StackWithIntrinsicChildren-mount.html">1082</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/StackWithTextChildren-mount_base.html">3352</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/StackWithTextChildren-mount.html">3385</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/SwatchColorPicker-mount_base.html">7298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/SwatchColorPicker-mount.html">7364</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/TagPicker-mount_base.html">1729</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/TagPicker-mount.html">1731</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Text-mount_base.html">484</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Text-mount.html">482</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/TextField-mount_base.html">1138</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/TextField-mount.html">1138</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ThemeProvider-mount_base.html">999</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ThemeProvider-mount.html">1018</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ThemeProvider-virtual-rerender_base.html">783</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ThemeProvider-virtual-rerender.html">794</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1541</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Toggle-mount_base.html">775</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/Toggle-mount.html">787</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/buttonNative-mount_base.html">236</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27740/perf-test/buttonNative-mount.html">243</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: change/@fluentui-babel-preset-global-context-ebd46a3d-588a-4e6a-9981-389528ddb6f3.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-babel-preset-global-context-ebd46a3d-588a-4e6a-9981-389528ddb6f3.json:1]: ## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27740&commitId=bbf0c6e55ae5763cd8297da1bd80b41dcea3ec61&env=prod)

✅ There was 0 screenshots added, 1 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Size.chromium.png|0| Removed|

</details>


<div id="vrtCommentFluentUI-v0"/>

**Review by ling1726 — APPROVED** [review]

**Review by Hotell — APPROVED** [review]

**Review by sopranopillow — APPROVED** [review]

**Review by behowell — APPROVED** [review]

**Comment by Hotell** [comment]
feel free to admin merge once pipeline passes - this is important fix for upcoming release and you got already big enough approvals cross team. ty

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

**Comment by layershifter** [comment]
> feel free to admin merge once pipeline passes - this is important fix for upcoming release and you got already big enough approvals cross team. ty

Force merged, thanks 👍

**Pull request merged**